recipe/meta.yaml

```diff
@@ -1,41 +1,42 @@
-{% set version = "0.14.1" %}
+{% set version = "0.14.2" %}
 
 package:
   name: statsmodels
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/s/statsmodels/statsmodels-{{ version }}.tar.gz
-  sha256: 2260efdc1ef89f39c670a0bd8151b1d0843567781bcafec6cda0534eb47a94f6
+  sha256: 890550147ad3a81cda24f0ba1a5c4021adc1601080bd00e191ae7cd6feecd6ad
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
-  skip: true  # [py<37]
+  skip: true  # [py<39]
+  skip: true  # [python_impl == "pypy"]
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython >=0.29.26,<3                    # [build_platform != target_platform]
+    - cython >=3.0.10,<4                     # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
     - {{ compiler('c') }}
+    - {{ stdlib("c") }}
   host:
     - python
     - pip
-    - cython >=0.29.26,<3
+    - cython >=3.0.10,<4
     # Do not add any constraint on numpy here to use global pinning
     - numpy
-    - scipy >=1.4
-    - pandas >=1.0
-    - patsy >=0.5.2
-    - setuptools_scm >=7.0.0,<7.1.dev0
+    - scipy >=1.13.0
```

Comment:
That said, scipy 1.4 -> 1.13 is quite the jump, haha (almost 5 years worth of releases). Nothing wrong with it per se, just pointing it out. :)

Reply:
The 1.13 is needed since it is the only build with a NumPy 2 support, and we need binary information from SciPy.  We have used this pin for building (only) on PyPI.  The requirement is 1.8 for running (running on SciPy < 1.13 means using NumPy 1.x).

Reply:
This lower bound is not necessary, because it would have been enforced automatically through the (non-)availability of the respective builds - and for the PyPy builds, older numpy/scipy would have sufficed.